tensorflow/core/kernels/depthwise_conv_op.cc

```diff
@@ -376,7 +380,12 @@ class DepthwiseConv2dNativeOp : public BinaryOp<T> {
     // TODO(csigg): Have autotune decide if native is faster than cuDNN.
     // If in_depth==1, this operation is just a standard convolution.
     // Depthwise convolution is a special case of cuDNN's grouped convolution.
-    bool use_cudnn = use_cudnn_ && (in_depth == 1 || use_cudnn_grouped_conv_);
+    // Use CuDNN grouped conv when filter is 1x1, 3x3, 5x5, 7x7.
+    // See cudnn release note 7.6.3. (https://docs.nvidia.com/deeplearning/sdk/c
+    // udnn-release-notes/rel_763.html#rel_763)
+    bool use_cudnn = use_cudnn_ && (in_depth == 1 || (use_cudnn_grouped_conv_ &&
+        filter_rows == filter_cols && (filter_rows == 1 || filter_rows == 3 ||
+        filter_rows == 5 || filter_rows == 7)));
```

Comment:
Let's factor this out into a `IsCudnnSupportedFilterSize()`.

Reply:
Do you think we need to put this function to `tensorflow/core/util/use_cudnn.h` and then both depthwise_conv_op.cc and depthwise_conv_grad_op.cc can share it? Or do we just duplicate it in both files?

Reply:
Putting this in `tensorflow/core/util/use_cudnn.h ` sounds great!

Reply:
Thanks for pointing out. Done. PTAL.